File: classfields/TestActivity.java
package com.yandex.mobile.verticalapp;

import android.content.Intent;
import android.os.Bundle;
import androidx.annotation.Nullable;
import android.view.View;
import com.yandex.mobile.verticalapp.dynamicscreen.DynamicScreenActivity;
import com.yandex.mobile.verticalwidget.activity.BaseCoreActivity;

/**
 * @author ironbcc on 22.04.2015.
 */
public class TestActivity extends BaseCoreActivity implements View.OnClickListener {

    @Override
    public void onCreate(@Nullable Bundle savedInstanceState) {
        super.onCreate(savedInstanceState);
        setContentView(R.layout.activity_all_widgets);
        ((TestApplication) getApplication()).getAppComponent().inject(this);
        findViewById(R.id.dynamic_screen_btn).setOnClickListener(this);
    }

    @Override
    public void onClick(View v) {
        switch (v.getId()) {
            case R.id.dynamic_screen_btn:
                startActivity(new Intent(this, DynamicScreenActivity.class));
                break;
        }
    }
}

File: mobile/test/java/ru/yandex/yandexmaps/common/conductor/TestActivity.kt
package ru.yandex.yandexmaps.common.conductor

import android.app.Activity
import android.os.Bundle
import android.widget.FrameLayout
import com.bluelinelabs.conductor.Conductor
import com.bluelinelabs.conductor.Router

const val ROOT_ID = 42

open class TestActivity : Activity() {

    var bundle: Bundle? = null

    lateinit var router: Router

    override fun onCreate(savedInstanceState: Bundle?) {
        bundle = savedInstanceState
        super.onCreate(savedInstanceState)

        val root = FrameLayout(this)
        root.id = ROOT_ID
        setContentView(root)

        router = Conductor.attachRouter(this, this.findViewById(ROOT_ID), savedInstanceState)
    }
}
